docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -132,42 +200,41 @@ the most popular are
 To be able to view the arguments as passed by Ansible to the module follow
 these steps.
 
-- Prefix the Ansible command with :envvar:`ANSIBLE_KEEP_REMOTE_FILES=1` to get Ansible to keep the exec files on the server
-- Log onto the Windows server using the same user Ansible executed the module as
-- Navigate to ``%TEMP%\..``, there should be a folder starting with ``ansible-tmp-``
-- Inside this folder open up the powershell script for the module
-- In this script there is a raw JSON script under ``$json_raw`` which contains the module arguments under ``module_args``
-- These args can be assigned manually to the ``$complex_args`` variable that is defined on your debug script
+- Prefix the Ansible command with :envvar:`ANSIBLE_KEEP_REMOTE_FILES=1` to specify that Ansible should keep the exec files on the server.
+- Log onto the Windows server using the same user account that Ansible used to execute the module.
+- Navigate to ``%TEMP%\..``. It should contain a folder starting with ``ansible-tmp-``.
+- Inside this folder, open the PowerShell script for the module.
+- In this script is a raw JSON script under ``$json_raw`` which contains the module arguments under ``module_args``. These args can be assigned manually to the ``$complex_args`` variable that is defined on your debug script.
 
 
 Windows unit testing
 ====================
 
 Currently there is no mechanism to run unit tests for Powershell modules under Ansible CI.
-There is work in the pipeline to introduce this in the future, stay tuned.
```

Comment:
Removed. Please do not discuss possible upcoming features in the documentation. Feel free to ping me for further explanation.